src/assets/locales/en.js

```diff
@@ -556,6 +556,8 @@ export default {
       num_questions_answered: {
         title: "QUESTIONS ANSWERED",
       },
+      survey_warning:
+        "*Survey mode questions are not considered in the calculation of any metric",
```

Comment:
`*` is common across all languages right? - best to move it outside this string and include it in the html itself.